pkgs/applications/system/asusctl/default.nix

```diff
@@ -0,0 +1,102 @@
+    sed -i -e s,/usr/bin/chattr,${e2fsprogs}/bin/chattr, daemon/src/ctrl_platform.rs
+  '';
+
+  postInstall = ''
+    install -Dm644 $src/data/asusd.conf $out/share/dbus-1/system.d/asusd.conf
+    install -Dm644 $src/rog-aura/data/layouts/g513_US.toml $out/share/rog-gui/layouts/g513_US.toml
+    install -Dm644 $src/rog-aura/data/layouts/g533_US.toml $out/share/rog-gui/layouts/g533_US.toml
+    install -Dm644 $src/rog-aura/data/layouts/ga401_US.toml $out/share/rog-gui/layouts/ga401_US.toml
+    install -Dm644 $src/rog-aura/data/layouts/gl504_US.toml $out/share/rog-gui/layouts/gl504_US.toml
+    install -Dm644 $src/rog-aura/data/layouts/gx502_US.toml $out/share/rog-gui/layouts/gx502_US.toml
+
+    install -Dm644 $src/rog-control-center/data/rog-control-center.desktop $out/share/applications/rog-control-center.desktop
+    install -Dm644 $src/rog-control-center/data/rog-control-center.png $out/share/icons/hicolor/512x512/apps/rog-control-center.png
+
+    install -Dm644 $src/data/icons/asus_notif_yellow.png $out/share/icons/hicolor/512x512/apps/asus_notif_yellow.png
+    install -Dm644 $src/data/icons/asus_notif_green.png $out/share/icons/hicolor/512x512/apps/asus_notif_green.png
+    install -Dm644 $src/data/icons/asus_notif_red.png $out/share/icons/hicolor/512x512/apps/asus_notif_red.png
+
+    install -Dm644 $src/data/icons/scalable/gpu-compute.svg $out/share/icons/hicolor/scalable/status/gpu-compute.svg
+    install -Dm644 $src/data/icons/scalable/gpu-hybrid.svg $out/share/icons/hicolor/scalable/status/gpu-hybrid.svg
+    install -Dm644 $src/data/icons/scalable/gpu-integrated.svg $out/share/icons/hicolor/scalable/status/gpu-integrated.svg
+    install -Dm644 $src/data/icons/scalable/gpu-nvidia.svg $out/share/icons/hicolor/scalable/status/gpu-nvidia.svg
+    install -Dm644 $src/data/icons/scalable/gpu-vfio.svg $out/share/icons/hicolor/scalable/status/gpu-vfio.svg
+    install -Dm644 $src/data/icons/scalable/notification-reboot.svg $out/share/icons/hicolor/scalable/status/notification-reboot.svg
+    install -Dm644 $src/data/icons/scalable/notification-reboot.svg $out/share/icons/hicolor/scalable/status/notification-reboot.svg
+
+    install -Dm644 $src/rog-anime/data/anime/asus/rog/Sunset.gif $out/share/asusd/anime/asus/rog/Sunset.gif
+    install -Dm644 $src/rog-anime/data/anime/custom/sonic-run.gif $out/share/asusd/anime/custom/sonic-run.gif
+    install -Dm644 $src/rog-anime/data/anime/custom/sonic-wait.gif $out/share/asusd/anime/custom/sonic-wait.gif
+    install -Dm644 $src/rog-anime/data/anime/custom/rust.png $out/share/asusd/anime/custom/rust.png
+    install -Dm644 $src/rog-anime/data/anime/custom/diagonal-template.png $out/share/asusd/anime/custom/diagonal-template.png
+
+    install -Dm644 $src/rog-anime/data/anime/asus/gaming/Controller.gif $out/share/asusd/anime/asus/gaming/Controller.gif
+
+    install -Dm644 $src/data/asusd.rules $out/lib/udev/rules/99-asusd.rules
```

Comment:
Is there no install script? Can we shrink this with wildcards or functions?

Reply:
Factorized using *.
The makefile is not usable because they install other things like the service and rules. I think it is better to do this in the nix file instead of patching the makefile.